src/digraph/Dominator_Tree.jl

```diff
@@ -0,0 +1,330 @@
+function dominator_Tree(g::AG, source) where {T, AG<:DiGraph{T}}
```

Comment:
This needs a docstring.

Reply:
i thought it is better to wait until we settle on the final code, and then write the docsring